### PR TITLE
Update README to mention all pug alias options

### PR DESCRIPTION
**Pull request by lehni**
As discussed here: https://github.com/prettier/plugin-pug/issues/118#issuecomment-700724677

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -200,7 +202,8 @@ They should be set via `Prettier`'s `overrides` option
   - `'trim-all'` -> Trim all spaces within comments.  
     Example: `// this is a comment`
 
-- `pugSortAttributesBeginning` (and `pugSortAttributesEnd`)
+- `sortAttributesBeginning` | `pugSortAttributesBeginning`
```
Comment (Shinigami92) [inline README.md:205]: There is (intentionally) no `sortAttributesBeginning` or `sortAttributesEnd`
I also think about to deprecate all previous options without `pug` prefix that are not a prettier default option

**Review by Shinigami92 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (lehni) [reply README.md:205]: Ah right, you mentioned that! Will remove it. Shall I remove the other also, so people stop using them?

Comment (Shinigami92) [reply README.md:205]: I think yes, we should remove the non alias ones (pug-specific-option)

Comment (lehni) [reply README.md:205]: Ok, change coming up!

Comment (lehni) [reply README.md:205]: But then the whole part about __They should be set via `Prettier`'s `overrides` option__ can be left out, right? Making docs simpler, a good thing!

Comment (Shinigami92) [reply README.md:205]: Yes, this also can be upgraded to just use the new pug prefixes

You only need to mention specifically that `singleQuote` will be the opposite of what prettier attributes will be quoted
Cause the js code in values of pug attributes uses these quotes

Comment (lehni) [reply README.md:205]: Yes makes sense!

**Review by Shinigami92 — APPROVED** [review]

**Comment by lehni** [comment]
Thanks for the merge, and apologies for the spelling mistakes that made it through 🙈

**Pull request merged**